gui/src/mainwindow.ui

```diff
@@ -24,11 +24,8 @@
     <normaloff>:/Icons/images/Icon256.png</normaloff>:/Icons/images/Icon256.png</iconset>
   </property>
   <widget class="QWidget" name="centralwidget">
-   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,2">
-    <property name="sizeConstraint">
-     <enum>QLayout::SetDefaultConstraint</enum>
-    </property>
-    <item>
+   <layout class="QGridLayout" name="gridLayout_14">
```

Comment:
yeah, this might be my bug, I may need to start again to get a Box Layout here...

Reply:
yeah, this is cleaned up now